packages/discovery-react-components/src/utils/document/processDoc.ts

```diff
@@ -67,7 +71,7 @@ export interface Table {
  * Convert document data into structure that is more palatable for use by
  * CIDocument
  *
- * @param {Object} data Discovery document data
+ * @param {Object} query Discovery document data
```

Comment:
I don't like this change. `query` implies something else. If `data` didn't work, then perhaps `queryResult` or `queryData`.

Reply:
queryData it is 👍

Reply:
Fixed https://github.com/watson-developer-cloud/discovery-components/pull/13/commits/f7415ec528350185a2ea54d0dfe82c468f2a796e